resources/assets/js/home.js

```diff
@@ -393,9 +393,14 @@ window.PoliceScorecardHome = window.PoliceScorecardHome || (function () {
             }, 'admin-0-country-borders');
 
             // Join the sheriff data with the lookup data based on FIPS code
-            function setCounties(e) {
+            function setCounties() {
+                if (!lookupData || !sheriffData || typeof sheriffData.features === 'undefined') {
+                    return false;
+                }
+
                 sheriffData.features.forEach(function(row) {
-                    if (row.properties.fips < 46000 || row.properties.fips > 47000) {
+                    // validate Mapping of FIPS data before using it
+                    if (typeof row.properties !== 'undefined' && typeof row.properties.fips !== 'undefined' && typeof lookupData[row.properties.fips] !== 'undefined') {
```

Comment:
Extract into variable for clarity and to avoid comment?

```suggestion
                    const hasFipsData = typeof row.properties !== 'undefined' && typeof row.properties.fips !== 'undefined' && typeof lookupData[row.properties.fips] !== 'undefined';
                    if (hasFipsData) {
```

Reply:
I am fine with this, but I personally do not like making one time use variables, especially since it causes memory bloat.  Could just be a comment above the conditional IMO if we are looking to avoid confusion.  Thoughts?